lib/Cleantalk/Common/Firewall/FirewallUpdater.php

```diff
@@ -215,6 +224,9 @@ public function update()
                         //Files array is empty update sfw stats
                         $helper::SfwUpdate_DoFinisnAction();
 
+                        if ( $fw_stats['firewall_subnets_count'] == $this->getSfwBlacklistsCount($this->api_key) ) {
```

Comment:
This additional call of the API method `get_2s_blacklists_db` need to be removed!